From 2b6ea2efd209d7aa443225f5061d37bd3dde059d Mon Sep 17 00:00:00 2001
From: "Zhong,Fangjian" <fangjian.zhong@intel.com>
Date: Thu, 22 Jun 2023 18:49:38 +0800
Subject: [PATCH] Add sepolicy for bootcontrol service

New service vendor.boot-default created for bootcontrol, allow
launching specified application for service.

Tracked-On: OAM-110961
Signed-off-by: Zhong,Fangjian <fangjian.zhong@intel.com>
---
 slot-ab/file_contexts | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 slot-ab/file_contexts

diff --git a/slot-ab/file_contexts b/slot-ab/file_contexts
new file mode 100644
index 0000000..4327d98
--- /dev/null
+++ b/slot-ab/file_contexts
@@ -0,0 +1 @@
+/vendor/bin/hw/android\.hardware\.boot-service\.intel u:object_r:hal_bootctl_default_exec:s0
-- 
2.25.1

